src/napari_graph/base_graph.py

```diff
@@ -138,14 +138,14 @@ def _iterate_edges(
 
 
 @njit
-def _create_world2buffer_map(world_idx: np.ndarray) -> typed.Dict:
-    """Fills world indices to buffer indices mapping."""
-    world2buffer = typed.Dict.empty(types.int64, types.int64)
-
-    for i in range(world_idx.shape[0]):
-        world2buffer[world_idx[i]] = i
-
-    return world2buffer
+def _insert_world2buffer(
```

Comment:
Maybe call it `_init_world2buffer`, `_fill_world2buffer` or something like that, to emphasize the "mutable character" of the function? No strong opinions on my side, happy to keep the current name as well

Reply:
I'm not in favor of using `init` or `fill` because it doesn't reset the `world2buffer` dictionary.
`update` could be an alternative if you prefer.

Reply:
Happy with `update` - thanks!

Reply:
Done